### PR TITLE
Break out remaining routes that post to display map

### PR DESCRIPTION
**Pull request by dsimmons87**
Moves account blocked to its own route from "display map" so that it can be linked to if needed and so it is isolated for testing.

Moves post comment to its own route from "display map" to make it more easily testable and prevent form resubmission issues.

### CODE REVIEW

File: openra/tests/routes/test_route_maps_post_comment.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class TestRouteMapsPostComment(TestRouteBase):
     _route = '/maps/{}/post-comment'
 
     def test_route_can_be_accessed_by_any_user(self):
-        user = self.create_old_user();
+        user = self.create_old_user()
```
Comment (pchote) [inline openra/tests/routes/test_route_maps_post_comment.py:16]: Can these to fixes be squashed back into the first comment?

File: openra/tests/routes/test_route_user_actions_blocked.py [hunk 2]
```diff
@@ -53,7 +55,24 @@ def test_route_redirects_to_login_if_not_authed(self):
...
+
+        response = self.get_authed(
+            {},
+            user
+        )
+
+        self.assert_contains(
+            response,
+            [
+
```
Comment (pchote) [inline openra/tests/routes/test_route_user_actions_blocked.py:68]: Is this empty line intentional?

**Review by pchote — COMMENTED** [review]
Code changes all look reasonable, just a couple of minor nits. Not tested.

File: openra/tests/routes/test_route_maps_post_comment.py (hunk 1 above)
Comment (dsimmons87) [reply openra/tests/routes/test_route_maps_post_comment.py:16]: Sure, I've squashed them all up now

File: openra/tests/routes/test_route_user_actions_blocked.py (hunk 2 above)
Comment (dsimmons87) [reply openra/tests/routes/test_route_user_actions_blocked.py]: Nope, I've gotten rid of that now

**Review by pchote — APPROVED** [review]

**Pull request merged**